airbyte-ci/connectors_ci/metadata_service/orchestrator/README.md

```diff
@@ -0,0 +1,70 @@
+# Connector Orchestrator (WIP)
+This is the Orchestrator for Airbyte metadata built on Dagster.
+
+
+# Setup
+## Local
+All commands below assume you are in the `metadata_service/orchestrator` directory.
+### Installation
+```bash
+poetry install
+cp .env.template .env
+```
+
+### Create a GCP Service Account and Dev Bucket
+Developing against the orchestrator requires a development bucket in GCP.
+
+The orchestrator will use this bucket to:
+- store important output files. (e.g. Reports)
+- watch for changes to the `catalog` directory in the bucket.
+
+However all tmp files will be stored in a local directory.
+
+To create a development bucket:
+1. Create a GCP Service Account with the following permissions:
+    - Storage Admin
+    - Storage Object Admin
+    - Storage Object Creator
+    - Storage Object Viewer
+2. Create a GCS bucket
+3. Add the service account as a member of the bucket with the following permissions:
+    - Storage Admin
+    - Storage Object Admin
+    - Storage Object Creator
+    - Storage Object Viewer
+4. Add the following environment variables to your `.env` file:
+    - `METADATA_BUCKET`
+    - `GCP_GSM_CREDENTIALS`
+
+Note that the `GCP_GSM_CREDENTIALS` should be the raw json string of the service account credentials.
+
+Here is an example of how to import the service account credentials into your environment:
+```bash
+export GCP_GSM_CREDENTIALS=`cat /path/to/credentials.json`
+```
+
+### Start the orchestrator
+Start the orchestrator with the following command:
+```bash
+poetry run dagster dev -m orchestrator
```

Comment:
Good question, ill add the answer to the readme.

It will launch the Daggit UI and a few additional services but it wont compile your assets until you click in the UI.

Actually Ill add a section about compiling assets in a headless way (CLI only)